meganav.yaml

```diff
@@ -0,0 +1,951 @@
+              description: Discuss Ubuntu with other users
+              url: https://ubuntuforums.org/
+            - title: Ubuntu Masters
+              description: Get inspired by IT practitioners
+              url: /masters-conference
+
+      secondary_links:
+        - title: Quick links
+          links:
+            - title: Upgrade Ubuntu
+              url: /tutorials/upgrading-ubuntu-desktop
+            - title: Official Ubuntu documentation
+              url: https://help.ubuntu.com/
+            - title: Install guide
+              url: /tutorials/install-ubuntu-desktop
+            - title: How to write a tutorial
+              url: /tutorials/tutorial-guidelines
+
+    - title: Forums
+      primary_links:
+        - links:
+            - title: Ask Ubuntu
+              description: Q&A for users and developers
+              url: https://askubuntu.com/
+            - title: Forums
+              description: Discuss Ubuntu with other users
+              url: https://ubuntuforums.org/
+            - title: Chat on IRC
+              description: Speak to Ubuntu users
+              url: https://help.ubuntu.com/community/InternetRelayChat
+            - title: Discourse
+              description: Coordinating the Ubuntu project
+              url: https://discourse.ubuntu.com/
+
+      secondary_links:
+        - title: Quick links
+          links:
+            - title: "Forum: New to Ubuntu"
+              url: https://ubuntuforums.org/forumdisplay.php?f=125
+            - title: "Forum: Ubuntu Official Flavours Support"
+              url: https://ubuntuforums.org/forumdisplay.php?f=327
+            - title: "Forum: Ubuntu, Linux and OS Chat"
+              url: https://ubuntuforums.org/forumdisplay.php?f=434
+            - title: Ubuntu Community on Discourse
+              url: https://discourse.ubuntu.com/c/community/78
+
+    - title: Contribute to Ubuntu
+      primary_links:
+        - links:
+            - title: Translate and localise
+              description: Translate into your language
+              url: https://ubuntu.com/community/contribute/translation-localisation
+            - title: QA and testing
+              description: Test releases and features
+              url: https://ubuntu.com/community/contribute/qa-testing
+            - title: Report bugs
+              description: Report bugs on Launchpad
+              url: https://help.ubuntu.com/community/ReportingBugs
+            - title: Software development
+              description: Create and maintain software
+              url: https://ubuntu.com/community/contribute/software-development
+            - title: Improve documentation
+              description: Help write the docs
+              url: https://wiki.ubuntu.com/DocumentationTeam/SystemDocumentation
+            - title: Art and design
+              description: Contribute your artwork
+              url: https://ubuntu.com/community/contribute/art-design
```

Comment:
The ubuntu.com links here should be relative.

Reply:
Fixed! Thank you Ant